src/themes/default/components/core/blocks/SidebarMenu/SubCategory.vue

```diff
@@ -144,7 +144,7 @@ export default {
   },
   methods: {
     logout () {
-      this.$bus.$emit('user-before-logout')
+      this.$store.dispatch('user/logout', {})
```

Comment:
couldn't be just
```suggestion
      this.$store.dispatch('user/logout')
```
?
Accepting either way :)

Reply:
it causes error because of https://github.com/DivanteLtd/vue-storefront/blob/develop/core/modules/user/store/actions.ts#L213 (requires object)